src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,8 +5,6 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
-    public Bulldozer() {
-    }
```

Comment:
* Add some custom fields that could be only in Bulldozer
* Do not remove no-args constructor